package.json

```diff
@@ -1101,12 +1101,12 @@
     "@babel/generator": "^7.21.1",
     "@babel/helper-plugin-utils": "^7.20.2",
     "@babel/parser": "^7.21.2",
-    "@babel/plugin-proposal-class-properties": "^7.18.6",
     "@babel/plugin-proposal-export-namespace-from": "^7.18.9",
     "@babel/plugin-proposal-nullish-coalescing-operator": "^7.18.6",
     "@babel/plugin-proposal-object-rest-spread": "^7.20.7",
     "@babel/plugin-proposal-optional-chaining": "^7.21.0",
     "@babel/plugin-proposal-private-methods": "^7.18.6",
+    "@babel/plugin-transform-class-properties": "^7.23.3",
```

Comment:
The babel package `@babel/plugin-proposal-class-properties` was also switched for `@babel/plugin-transform-class-properties` as the former is now deprecated.